doc/manuals/user/walkthrough_apiv1.md

```diff
@@ -127,13 +127,13 @@ procedure](../admin/database_admin.md#delete-complete-database).
 To start the broker (as root or using the sudo command):
 
 ```
-/etc/init.d/contextBroker start
+/etc/init.d/contextBroker start
```

Comment:
Don't know why github marks this as change... I haven't (consciously) touched this part.

NTC